src/pages/billing/Project.js

```diff
@@ -361,19 +396,32 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
                   value: days
                 }), [7, 30, 90]),
                 onChange: ({ value: selectedDays }) => {
-                  setSpendReportLengthInDays(selectedDays)
-                  setProjectCost(null)
+                  if (selectedDays !== spendReportLengthInDays) {
```

Comment:
@msilva-broad This is a pre-existing bug that was bothering me so I pushed up a commit to fix it. Selecting an already-selected date range caused the cost cards to all clear and never reload.